packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -119,16 +109,12 @@ export function useSpecificEditorSettings() {
 			const _context = getEditedPostContext();
 			return {
 				templateSlug: _record.slug,
-				isDistractionFree: !! getPreference(
-					'core/edit-site',
-					'distractionFree'
-				),
 				canvasMode: getCanvasMode(),
 				settings: getSettings(),
 				postWithTemplate: _context?.postId,
 			};
 		},
-		[ isLargeViewport ]
```

Comment:
Looks like this was only used for the top toolbar setting, which got moved to core by https://github.com/WordPress/gutenberg/pull/57531/. Good catch on removing this.